src/db_api_util.ts

```diff
@@ -86,6 +86,184 @@ export type DatabaseCommandEntry = {
   failureReason?: string | null;
 };
 
+export type DatabaseThreadEntry = {
```

Comment:
I have this file ordered such that the functions associated with tables that are most important (e.g. messages) are at top. Can you move these functions to the bottom?